clusters/nerc-ocp-prod/griot-grits/application-set.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: griot-and-grits-apps
+spec:
+  syncPolicy:
+    preserveResourcesOnDeletion: false
+  generators:
+  - git:
+      repoURL: https://github.com/OCP-on-NERC/griot-and-grits-infra.git
+      revision: main
+      files:
+      - path: applications/*/kustomization.yaml
+  template:
+    metadata:
+      name: '{{path.basename}}'
+      labels:
+        app.kubernetes.io/name: '{{path.basename}}'
+        app.kubernetes.io/component: griot-and-grits
+        app.kubernetes.io/part-of: griot-and-grits
+        nerc.mghpcc.org/sync-policy: common
+    spec:
+      project: default
+      source:
+        repoURL: https://github.com/griot-and-grits/griot-and-grits-infra.git
```

Comment:
Also just want to make sure the different URLs for the generator and source are intentional. So as written it will scan THIS repo (OCP-on-NERC/griot-and-grits-infra) for every file that matches applications/*/kustomization.yaml and create a set of variables (path, path.dir, etc.) for each match. Then when it  actually uses the values from the generator, it will apply the manifests the come from THIS repo (griot-and-grits/griot-and-grits-infra)